code/modules/uplink/uplink_items.dm

```diff
@@ -414,8 +414,8 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 /datum/uplink_item/dangerous/rapier
 	name = "Rapier"
 	desc = "A fancy rapier with a diamond tip piercing anything that it comes into contack with. \
-			The rapier comes with its own shielf, this is rather noticeable as only the captain is known to carry a shielf. \
-			The shielf itself can be used to block melee attacks only. Its also jet black colours."
+			The rapier comes with its own sheath, this is rather noticeable as only the captain is known to carry a sheath. \
```

Comment:
You miss spelled Shielf 4 times in a row man

Reply:
I don't think shielf is a word

Reply:
IT IS DAMN IT